suite-native/module-send/src/components/SendFeesForm.tsx

```diff
@@ -134,11 +138,13 @@ export const SendFeesForm = ({ accountKey, tokenContract }: SendFormProps) => {
             {/*// BottomSheetModalProvider must be inside FormProvider to keep context available for sheets rendered via portal*/}
             <BottomSheetModalProvider>
                 <VStack spacing="sp32" flex={1}>
-                    <RecipientsSummary
-                        accountKey={accountKey}
-                        tokenContract={tokenContract}
-                        selectedFeeLevel={selectedFeeLevelTransaction}
-                    />
+                    {selectedFeeLevelTransaction && (
```

Comment:
`selectedFeeLevelTransaction` is not a boolean
```suggestion
                    {!!selectedFeeLevelTransaction && (
```

Reply:
updated in 036bd6f17b6